### ISSUE TITLE
Trying to run gvenzl/Oracle-XE image From a Synology 920 DSM Container 

### ISSUE THREAD

**Issue by dbreslin42** [gvenzl/oci-oracle-xe#242]
Hi.
I'll start off by saying I am totally new to container-based executions but I'm willing to experiment and learn as I'm trying to run an Oracle Database from my NAS unit so that the overhead isn't on my main laptop. Based on my research this seems to be the a version of the DB that is designed for container use (and containers are the only way it seems I can run the database on the DSM OS)

I was able to locate this image in the Docker Registry and downloaded it.
Now for anyone unfamiliar with the  Container Manager in Synology, its GUI Based and so I am unable to edit anything or execute a terminal until I have a stable container open then I get a terminal option.

When I create the container, this is what the GUI looks like
![ss1](https://github.com/gvenzl/oci-oracle-xe/assets/155678807/e9fc0705-0a42-4414-a3f9-c60b637b2a47)
![ss2](https://github.com/gvenzl/oci-oracle-xe/assets/155678807/7df24925-fb11-406a-b4a7-a89a28c3b992)
![ss3](https://github.com/gvenzl/oci-oracle-xe/assets/155678807/b99b7e32-f6ed-4a33-80e2-b531a7c66d93)
![ss4](https://github.com/gvenzl/oci-oracle-xe/assets/155678807/a1f5b5cc-5dc5-4479-ab8b-ac5583bc30e0)

When I run the container script I receive the following error and have not been able to resolve it.
"Container starting up
Oracle Database SYS and SYSTEM passwords have to be specified at first database startup.
Please Specific a password either via the $ORACLE_PASSWORD variable, e.g. '-e ORACLE_PASSWORD=<password>' or set the $ORACLE_RANDOM_PASSWORD environment variable to any value, e.g. '-e ORACLE_RANDOM_PASSWORD=yes'.

I've tried to enter various derivatives of ORACLE_PASSWORD in the variable section and some sample code that I cant locate at the moment in the execution command section without success. Oh and executing this container dosn't even bring up a prompt or terminal automatically for me to enter the passwords. It simply closes the container with the above error.

Can anyone help?  I've found terminal based assistance with the Oracle documentation and general container based assistance for synology but no posts covering this image and a container in Synology; s environment.

Thanks in advance for ANY help for this middle-aged dog that is trying to learn new tricks in his free time to maintain being somewhat competitive at work

**Comment by gvenzl** [comment]
Hey @dbreslin42,

Thanks a lot for your interest in these images!

Hm, you should really not need all these variables to run the container, they are already all set in the container image itself. However, you will need the `ORACLE_PASSWORD` or `ORACLE_RANDOM_PASSWORD` variable set.
But you said even when that is set, the container still tells you that it isn't, is that right?

The container will never prompt you for the password as it assumes to be run in the background on some server. That's why the container stops if the variable isn't provided.

Perhaps you'd like to first try to run the container on your laptop? That way it would perhaps be easier to see the difference on the container manager software on the Synology DSM.

**Comment by dbreslin42** [comment]
Thank you for responding. I spent a lot of time trying to play with containers and getting it running on the NAS and the laptop and gave up. Found a Synology MariaDB Package and had that up and running in like 5 minutes. with PHPAdmin
Unfortunately been banging my head a lot with doing non-windows applications lately and in the end a GUI is nice to let the package do the install/configuration.

BUt yes I tried to set the passwords but there was no construct that the package seemed to accept so I could not proceed
